okhttp/src/jvmMain/kotlin/okhttp3/Dispatcher.kt

```diff
@@ -15,9 +15,7 @@
  */
 package okhttp3
 
-import java.util.ArrayDeque
-import java.util.Collections
-import java.util.Deque
+import java.util.*
```

Comment:
Revert star import. Seems like we need kotlinter/ktlint setup in here!

Reply:
Sorry, new laptop.